### PR TITLE
Add initial recipe for glue-vispy-viewers

### PR DESCRIPTION
**Pull request by astrofrog**
This is a plugin package for glueviz, which is already in conda-forge

### CODE REVIEW

**Comment by astrofrog** [comment]
It looks like this is failing because VisPy is not available for Python 3.5 on Windows. Is there a correct way to disable that build for now? (specifically Windows and Python 3.5)

**Comment by kmuehlbauer** [comment]
Should be something like this:
```yaml
build:
  skip: True  # [win and py=35]
```

**Comment by astrofrog** [comment]
@kmuehlbauer - thanks! Let's give it a try.

**Comment by astrofrog** [comment]
@kmuehlbauer this seems to have caused issues on Circle CI:

```
Error: Invalid selector in meta.yaml line 15:
  skip: True  # [win and py=35]
```

is there a reference for the syntax?

**Comment by jakirkham** [comment]
Drop `=`.

File: recipes/glue-vispy-viewers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  fn: glue-vispy-viewers-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/g/glue-vispy-viewers/glue-vispy-viewers-{{version}}.tar.gz
+  md5: 507d396069cb2eb96fc2e0c3058f88c3
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win and py=35]
```
Comment (ocefpaf) [inline recipes/glue-vispy-viewers/meta.yaml]: I believe that should be `# [win and py35]`.

**Comment by astrofrog** [comment]
@jakirkham - thanks!

File: recipes/glue-vispy-viewers/meta.yaml (hunk 1 above)
Comment (astrofrog) [inline recipes/glue-vispy-viewers/meta.yaml]: Fixed!

**Comment by kmuehlbauer** [comment]
Ahh, sorry, I took this from a `py<35` example, and thought the wrong way

File: recipes/glue-vispy-viewers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - glue_vispy_viewers
+
+about:
+  home: https://github.com/glue-viz/glue-vispy-viewers
+  license: BSD 3-Clause
+  summary: 3D viewers for Glue
+
+extra:
+  recipe-maintainers:
+    - astrofrog
```
Comment (ocefpaf) [inline recipes/glue-vispy-viewers/meta.yaml]: @astrofrog this recipe seems to be good to go. I just recommend you to invite more maintainers to share the load. (Can be done later in the feedstock.)

Comment (astrofrog) [inline recipes/glue-vispy-viewers/meta.yaml]: Ok - will do! :)

**Comment by astrofrog** [comment]
@ocefpaf @jakirkham @kmuehlbauer - it looks like the Windows builds are completely missing from https://anaconda.org/conda-forge/glue-vispy-viewers - does this indicate an issue with the skipping of the build for Python 3.5, or does it just take a while for the Windows builds to appear?

**Comment by kmuehlbauer** [comment]
@astrofrog you might have a look at appveyor (follow the link) , but if the queue is quite full than it will take a while.

**Pull request merged**